app/package.json

```diff
@@ -20,7 +20,8 @@
   "scripts": {
     "build": "webpack --progress --config webpack.build.config.js",
     "dev": "webpack --watch --config webpack.build.config.js",
-    "test": "webpack --config webpack.test.config.js"
+    "build-test": "webpack --config webpack.test.config.js ",
```

Comment:
I would rename this back to test and rename the "test" below to "test-chrome-headless"